packages/@aws-cdk/aws-cloudwatch/lib/metric.ts

```diff
@@ -372,6 +373,15 @@ export interface NewAlarmProps {
    * @default true
    */
   actionsEnabled?: boolean;
+
+  /**
+   * The number of datapoints that must be breaching to trigger the alarm. This is used only if you are setting an "M
+   * out of N" alarm. In that case, this value is the M. For more information, see Evaluating an Alarm in the Amazon
+   * CloudWatch User Guide.
+   *
+   * @default ``evaluationPeriods``
+   */
+  datapointsToAlarm?: number;
```

Comment:
couldn't we inherit `AlarmProps` from `NewAlarmProps`?

Reply:
I though of it but it seems messy somehow. But really it's just because the name is "New<something>" I'll look into improving this :)